src/modules/spotify/spotify-track-handler.ts

```diff
@@ -93,6 +93,27 @@ export default class SpotifyTrackHandler {
     );
   }
 
+  /**
+   * Skips to the next track.
+   */
+  public async skipToNext() {
+    if (
+      !this.api.client ||
+      !this.playState ||
+      !this.playState.device ||
+      !this.playState.device.id
+    ) {
+      logger.trace('Cannot skip to next track, no Spotify user or device');
+      return;
+    }
+
+    try {
+      await this.api.client?.player.skipToNext(this.playState.device.id);
+    } catch (e) {
+      logger.warn(e);
```

Comment:
I think we want to return an HTTP 500, as Spotify did not skip the track.

Reply:
Not a service unavailable?

Reply:
![image](https://github.com/user-attachments/assets/c2be6ce5-9105-496f-9ce4-c80e9e960c58)
I would not call this a temporary state, as we are dependent on another service.

_(I am doubtful this API call will even work, as Aurora is also not allowed to pause/resume playback at GEWIS. It is worth a try though)_